pkg/rancher-desktop/config/settings.ts

```diff
@@ -21,7 +21,7 @@ const console = Logging.settings;
 // it will be picked up from the default settings object.
 // Version incrementing is for when a breaking change is introduced in the settings object.
 
-export const CURRENT_SETTINGS_VERSION = 5 as const;
```

Comment:
Dropping the guarantee here doesn't seem very useful. It doesn't seem to get us anything either.

Reply:
If you mean the `as const` part, that breaks PR 3890 where I try to import the constant. I'm fine if you can show me the fix for that (in PR 3890, not here where it doesn't break anything).

Reply:
The only thing I can find in #3890 that would cause an issue is in a test (which makes sense, because we're _explicitly_ breaking assumptions there); that can be bypassed with a cast to `(…) as any`.

Reply:
ok, I'll move the change from the 3747e branch here...